### ISSUE TITLE
Java 16 Record results in a typescript base class Record

### ISSUE THREAD

**Issue by lpandzic** [vojtechhabarta/typescript-generator#657]
```java
public record Account (Long id,
                       String name) {
}
```

results in 

```typescript
export class Record {
}

export class Account extends Record {
    id: number;
    name: string;
}
```

Record class shouldn't be generated in typescript.

**Comment by vojtechhabarta** [comment]
It is possible to exclude `java.lang.Record` class so generated output is just
``` typescript
interface Account {
    id: number;
    name: string;
}
```

**Comment by lpandzic** [comment]
Since this is a standard java type, shouldn't this be done inside the library automatically?

**Comment by vojtechhabarta** [comment]
@lpandzic you are right, it's similar to Java enums which also have special treatment.

**Comment by lpandzic** [comment]
Ok, thanks, I take this is fixed.

**Comment by vojtechhabarta** [comment]
Released in [v2.32.889](https://github.com/vojtechhabarta/typescript-generator/releases/tag/v2.32.889).
